docs/introduction.rst

```diff
@@ -0,0 +1,111 @@
+Introduction
+============
+
+The XAITK-Saliency toolkit implements a class of XAI algorithms known
+as `saliency algorithms`. The basic application pipeline is shown in Figure 1:
+
+.. figure:: figures/intro-fig-01.png
+
+*Figure 1: A basic AI pipeline.*
+
+In this scenario, an AI algorithm operates on an input (text, image,
+etc.) to produce some sort of output. Saliency algorithms build on
```

Comment:
maybe provide examples of output as well?

`output (classification, detection, etc.)`

Reply:
sounds good